ansible/playbooks/roles/repository/files/download-requirements/src/command/rpm.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from src.command.command import Command
+
+
+class Rpm(Command):
+    """
+    Interface for `rpm`
+    """
+
+    def __init__(self, retries: int):
+        super().__init__('rpm', retries)
+
+    def is_package_installed(self, package: str) -> bool:
+        """
+        Check if `package` is installed on the OS.
+
+        :param package: to be checked if installed
+        :returns: True - package installed, False - otherwise
+        """
+        args: List[str] = ['--query',
+                           '--quiet',
+                           f'{package}']
+
+        if self.run(args).returncode == 0:
+            return True
```

Comment:
Here it may be good to check what codes in which situations are returned by rpm, to not hide errors.